tests/market/test_day_ahead.py

```diff
@@ -0,0 +1,161 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from unittest.mock import patch, MagicMock
+
+import pytest
+from gsy_framework.constants_limits import GlobalConfig
+from gsy_framework.data_classes import Bid, Offer, Trade
+from pendulum import datetime
+
+from gsy_e.models.area import Area
+from gsy_e.models.market import GridFee
+from gsy_e.models.market.day_ahead import (DayAheadMarkets, DAY_AHEAD_MARKET_LENGTH_MINUTES,
+                                           DAY_AHEAD_HOUR_OF_ROTATION)
+from gsy_e.models.market.future import FutureMarketException
+from tests.market.test_future import count_orders_in_buffers
+
+DEFAULT_CURRENT_MARKET_SLOT = datetime(2022, 6, 13, 0, 0)
+
+
+@pytest.fixture(name="day_ahead_market")
+def active_day_ahead_market() -> DayAheadMarkets:
+    """Fixture for activated day-ahead market."""
+    orig_day_ahead_duration = GlobalConfig.DAY_AHEAD_DURATION_DAYS
+    orig_start_date = GlobalConfig.start_date
+    GlobalConfig.FUTURE_MARKET_DURATION_HOURS = 1
```

Comment:
The original value of this one should also be reverted as part of the teardown steps, right?